pytest/tests/mocknet/load_testing.py

```diff
@@ -0,0 +1,36 @@
+# Force each node to submit many transactions for
+# about 20 minutes. Monitor the block production time
+# stays consistent.
+
+import sys, time
+from rc import pmap
+
+from load_testing_helper import TIMEOUT
+
+sys.path.append('lib')
+import mocknet
+
+nodes = mocknet.get_nodes()
+
+print('INFO: Starting Load test.')
+
+print('INFO: Performing baseline block time measurement')
+(block_time, stdev) = mocknet.measure_average_block_time(nodes[-1])
+assert block_time < 1.01
+assert stdev < 0.1
+print('INFO: Baseline block time measurement complete')
+
+print('INFO: Setting remote python environments.')
+mocknet.setup_python_environments(nodes)
+print('INFO: Starting transaction spamming scripts.')
+mocknet.start_load_test_helpers(nodes)
+
+start_time = time.time()
+while time.time() - start_time < TIMEOUT:
+    (block_time, stdev) = mocknet.measure_average_block_time(nodes[-1])
+    assert block_time < 1.01
+    assert stdev < 0.1
+
+    # TODO: other metrics
```

Comment:
@SkidanovAlex @bowenwang1996 

I'm looking for suggestions of what else would be good to measure here and how I should go about measuring it. I know we mentioned memory usage before. I could run `ps aux` on the machines to see the usage of the `near` process; does that sound sufficient? What should the bounds be in that case?